docs/custom-resource-sample/report_overview1.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: "insights.redhat.com/v1alpha"
+kind: ReportOverview
```

Comment:
Isn't this too general? Wouldn't be better e.g `InsightsReportOverview`? Just asking.... :-)